containers/app/Dockerfile

```diff
@@ -46,6 +46,13 @@ RUN mkdir -p $WORKSPACE_BASE
 RUN apt-get update -y \
     && apt-get install -y curl ssh sudo
 
+# Install Docker
+RUN apt install -y apt-transport-https ca-certificates gnupg \
+    && curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker.gpg] https://download.docker.com/linux/debian bookworm stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null \
+    && apt update \
```

Comment:
is `apt-update` needed twice?

Reply:
I was following this: https://linuxiac.com/how-to-install-docker-on-debian-12-bookworm/
But I can remove those and see how it goes. The `apt update` may be needed after we add that source but I'm not linux expert lol. Will try a few things

Reply:
@tobitege followed the official docs now: https://docs.docker.com/engine/install/debian/

Reply:
Yup after you add a new source you need to update